pkg/arvo/lib/hood/drum.hoon

```diff
@@ -250,7 +252,22 @@
     =<  (se-born %home %link-store)
     =<  (se-born %home %link-proxy-hook)
     =<  (se-born %home %link-listen-hook)
-    (se-born %home %link-view)
+    =<  (se-born %home %link-view)
+    (se-born %home %s3-store)
+  ::
+      %3
+    =<  se-abet  =<  se-view
+    =<  (se-emit %pass /kiln %arvo %g %sear ~wisrut-nocsub)
```

Comment:
This... clears pending queues for... some planet under ~litzod? I see this was done for previous version transitions, but I have no idea why. Was it just a weird hack that got cargo-culted forward?

Reply:
See #2553.  Was used to resolve a particular issue on a couple galaxies, but can be removed now.